frontend/src/components/Step3.tsx

```diff
@@ -0,0 +1,204 @@
+import { Link } from "react-router-dom";
+
+import { Button } from "./Button.tsx";
+import styles from "./Step3.module.css";
+import { WorkExperienceForm } from "./index.ts";
+
+// Pass in Application's next function
+export type StepProps = {
+  next: () => void;
+};
+
+export const Step3: React.FC<StepProps> = ({ next }: StepProps) => {
+  // Checks if all Required Inputs are filled before moving on to next step
+  const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const form = event.target as HTMLFormElement;
+    if (!form.checkValidity()) {
+      return;
+    }
+    next();
+  };
+
+  return (
+    <>
+      <form id="step3-form" onSubmit={onSubmit}>
+        <div className={styles.formContainer}>
+          <hr />
+
+          <div className={styles.formSection}>
+            <WorkExperienceForm
+              sectionName="Current Company Info"
+              formInputs={[
+                {
+                  inputTitle: "Company Name",
+                  defaultMessage: "Add Company Name",
+                  inputType: "text",
+                },
+                {
+                  inputTitle: "Website",
+                  defaultMessage: "Add your Company's Website",
+                  inputType: "text",
+                },
+                {
+                  inputTitle: "Profession",
+                  defaultMessage: "Add Profession Here",
+                  inputType: "text",
+                },
+                {
+                  inputTitle: "Specialization",
+                  defaultMessage: "Add Specialization Here",
+                  inputType: "text",
+                },
+              ]}
+            />
+          </div>
+          <hr />
+          <div className={styles.formSection}>
+            <WorkExperienceForm
+              sectionName="Work Experience"
+              formInputs={[
+                {
+                  inputTitle: "Recent diversified design experience",
+                  defaultMessage: "Design Experience",
```

Comment:
place holder text for every field in the Work Experience as "Enter" at the begining.